File: Abdul Bari/Recursion/Factorial.cpp
#include<stdio.h>

int fact(int n){

    if(n==0)
        return 1;
    
    else
        return n*fact(n-1);

}


int main(){

    int f=5;
    printf("Fact of %d is %d", f , fact(f));
    return 0;

}

File: Kunal Kushwaha/Java_basics/src/com/shaunak/Factorial.java
package com.shaunak;


import java.util.Scanner;

public class Factorial{

    public static void main(String[] args) {
        System.out.println("Enter a number:\n");
        Scanner input = new Scanner(System.in);

        int num = input.nextInt();
        int inputNum=num;
        long fact=1;

        while(num != 0){
            fact = fact * num;
            num--;
        }


        System.out.println("Factorial of "+ inputNum +" is "+ fact);
    }


}
